index.html

```diff
@@ -48,4 +48,178 @@ <h2 class="hex connect">LET’S CONNECT</h2>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">2015</li>
+            </ul>
+            </div>
+            <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            </p>
+            <ul class="categories hex">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">See Project</button></a>
+            </div>
+            </div>
+            </section>
+            <section class="grid-items other-projects">
+            <img class="project-img" src="images/projects/project-4.png" alt="Project-1" />
+            <div class="details">
+            <div class="titles">
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <ul class="canopy">
+            <li>CANOPY</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">Back End Dev</li>
+            <li class="bullets"><span class="dot"></span></li>
+            <li class="o-li">2015</li>
+            </ul>
+            </div>
+            <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+            </p>
+            <ul class="categories hex">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">See Project</button></a>
+            </div>
+            </div>
+            </section>
+            </div>
+            <div class="about-myself">
+            <h2 class="about">About Myself</h2>
+            <p class="text">
+            Hello I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <h2 class="hex connect">LET’S CONNECT</h2>
+            <ul class="social-media">
+            <li clalss="socials">
+            <a href="#"><img src="images/socials/twitter.svg" /></a>
+            </li>
+            <li clalss="socials">
+            <a href="#"><img src="images/socials/Linkedin.svg" /></a>
+            </li>
+            <li clalss="socials">
+            <a href="#"><img src="images/socials/m.svg" /></a>
+            </li>
+            <li clalss="socials">
+            <a href="#"><img src="images/socials/Vector1.svg" /></a>
+            </li>
+            <li clalss="socials">
+            <a href="#"><img src="images/socials/Vector2.svg" /></a>
+            </li>
+            </ul>
+            <div class="see-project">
+            <a href="#"><button type="button" class="hex">Get my resume</button></a>
```

Comment:
- [ ] Please increase the font-size and font-weight of the get my resume button to make it look like the one in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=uQuKu3bekkovCjcN-0)

Figma Design:

<img width="294" alt="Screenshot 2023-03-25 at 5 01 18 AM" src="https://user-images.githubusercontent.com/44798044/227668585-c4179396-cf71-4670-b458-2f4666b7551f.png">

Your Design:

<img width="311" alt="Screenshot 2023-03-25 at 5 01 09 AM" src="https://user-images.githubusercontent.com/44798044/227668597-5988900a-7e08-4d56-84c5-76cd50adfe5b.png">